### ISSUE TITLE
My profile does not show any posts I made

### ISSUE THREAD

**Issue by junghyuneun** [Docile-Alligator/Infinity-For-Reddit#1432]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.4.1

### Where did you get the app from

Google Play

### Android version

Android 13

### Device model

Nothing Phone (1)

### First occurred

Today

### Steps to reproduce

1. Open left navigation drawer
2. Click Profile
3. It says "No posts found"
4. At first it only showed 1 post (I have more than 10) then when I refreshed the page it shows what 3 says.
5. Somewhat unrelated but I can follow myself

### Expected behaviour

I should see all posts I made

### Current behaviour

Doesn't show my post history

**Comment by junghyuneun** [comment]
I see that it is marking my own posts as read and hiding it but I think it shouldn't happen in ANY profile pages?

**Comment by rasc42** [comment]
Yeah, posts are currently hidden from user profiles, which is a strange choice. There is currently a discussion for changing it here: https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/1007

But so few upvotes, since not that many people actually look into the discussions page, so it's been there for 3 years now =/
